### PR TITLE
Revisions for new CA renewal

### PR DESCRIPTION
**Pull request by tombentley**
### Type of change

- Documentation

### Description

Revise documentation for new-style CA renewal.

### Checklist

_Please go through this checklist and make sure all applicable tasks have been done_

- [ ] Write tests
- [ ] Make sure all tests pass
- [ ] Update documentation
- [ ] Check RBAC rights for Kubernetes / OpenShift roles
- [ ] Try your changes from Pod inside your Kubernetes and OpenShift cluster, not just locally
- [ ] Reference relevant issue(s) and close them after merging
- [ ] Update CHANGELOG.md

### CODE REVIEW

File: documentation/book/con-certificate-renewal.adoc [hunk 1]
```diff
@@ -35,24 +33,22 @@ The behavior of the Cluster Operator during the renewal period depends on whethe
 
 The Cluster Operator performs the following process to renew CA certificates:
 
-. Generate a new CA key and certificate. The new private key replaces the old private key in the corresponding `Secret`. The new certificate is given the name `ca.crt` within the corresponding `Secret` and the old certificate is renamed `ca-_<expiry-date>_.crt`.
+. Generate a new CA certificate, but retaining the existing key. The new certificate replaces the old one with the name `ca.crt` within the corresponding `Secret`.
 
-. Restart Zookeeper nodes so that they will trust the new CA certificate.
+. Generate new client certificates (for Zookeeper nodes, Kafka brokers, and the entity operator).
```
Comment (ppatierno) [inline documentation/book/con-certificate-renewal.adoc]: "Entity Operator" as for the others

File: documentation/book/con-certificate-renewal.adoc [hunk 2]
```diff
@@ -35,24 +33,22 @@ The behavior of the Cluster Operator during the renewal period depends on whethe
...
 
-. Perform the same restarts so that clients are using certificates signed by the new CA certificate.
-
-At the end of the renewal period the Cluster Operator will remove the now expired CA certificates (those named `ca-_<expiry-date>_.crt`) from the corresponding `Secret` and perform a further round of restarts.
 
 == Renewal process with your own CA certificates
 
-At the start of the renewal period the Cluster Operator will start logging at the `WARN` level that new CA certificates and keys are needed.
-Once you have provided the new certificates and keys, the Cluster Operator performs a further set of restarts within the Kafka cluster for which the warning was issued.
+At the start of the renewal period the Cluster Operator will start logging at the `WARN` level that new a CA certificate is needed.
```
Comment (ppatierno) [inline documentation/book/con-certificate-renewal.adoc]: typo ".. a new ... " not ".. new a ..."

File: documentation/book/con-certificate-renewal.adoc [hunk 3]
```diff
@@ -35,24 +33,22 @@ The behavior of the Cluster Operator during the renewal period depends on whethe
...
+Once you have provided the new certificate, the Cluster Operator performs a further set of restarts within the Kafka cluster for which the warning was issued.
```
Comment (ppatierno) [inline documentation/book/con-certificate-renewal.adoc]: Just  a doubt about this paragraph. You commented `proc-renewing-your-own-ca-certificates.adoc` but now we are leaving here how the renewal process works with your own CA certificates. I was wondering that if people read this, they could ask ... how to do that? and then the proc is needed.

Comment (tombentley) [reply documentation/book/con-certificate-renewal.adoc]: You're right. I guess I should just take this section out for the time being then.

Comment (ppatierno) [reply documentation/book/con-certificate-renewal.adoc]: Agree

**Pull request merged**